.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: php
 
+sudo: false
+
+git:
+  depth: 1
```

Comment:
Can you tell me more about why you decided to change this?  I agree that we probably don't need the default (50), but I'm wondering if 1 is too low:

> Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.

(https://docs.travis-ci.com/user/customizing-the-build#Git-Clone-Depth)

Reply:
Typically, if you don't do git operations you only need the depth of 1, saving a bit of resources when running the tests. IIUC this comes with the down side that when you commit a lot to a branch that Travis is testing not all commits are tested individually. As long as you are only interested in a PR with all the commits in it that doesn't matter, as long as you got one build on the latest commit :)

Reply:
:+1: for depth=1